curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --distractors--
 
-Placeholder distractor 1
+To change image size
 
 ---
 
-Placeholder distractor 2
+To add a title
 
 ---
 
-Placeholder distractor 3
+To change image color
 
 #### --answer--
 
-Placeholder answer
+To provide a text alternative
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is a screen reader?
 
 #### --distractors--
 
-Placeholder distractor 1
+A browser for disabled users
 
 ---
 
-Placeholder distractor 2
+A CSS tool
 
 ---
 
-Placeholder distractor 3
+A web development library
 
 #### --answer--
 
-Placeholder answer
+A text-to-speech software
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the 'Inspect Element' feature do?
 
 #### --distractors--
 
-Placeholder distractor 1
+Edits text directly
 
 ---
 
-Placeholder distractor 2
+Downloads files
 
 ---
 
-Placeholder distractor 3
+Modifies images
 
 #### --answer--
 
-Placeholder answer
+Displays CSS and HTML
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which CSS property controls text size?
 
 #### --distractors--
 
-Placeholder distractor 1
+text-size
 
 ---
 
-Placeholder distractor 2
+size-font
 
 ---
 
-Placeholder distractor 3
+text-style
 
 #### --answer--
 
-Placeholder answer
+font-size
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the opacity property do in CSS?
 
 #### --distractors--
 
-Placeholder distractor 1
+Changes text size
 
 ---
 
-Placeholder distractor 2
+Adds a border
 
 ---
 
-Placeholder distractor 3
+Sets the element's position
 
 #### --answer--
 
-Placeholder answer
+Controls element transparency
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which browser is known for robust developer tools?
 
 #### --distractors--
 
-Placeholder distractor 1
+Safari
 
 ---
 
-Placeholder distractor 2
+Internet Explorer
 
 ---
 
-Placeholder distractor 3
+Opera
 
 #### --answer--
 
-Placeholder answer
+Firefox
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is a common use for aria-hidden="true"?
 
 #### --distractors--
 
-Placeholder distractor 1
+To show content
 
 ---
 
-Placeholder distractor 2
+To display on mobile
 
 ---
 
-Placeholder distractor 3
+To increase visibility
 
 #### --answer--
 
-Placeholder answer
+To hide content from screen readers
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the flex property used for?
 
 #### --distractors--
 
-Placeholder distractor 1
+Changing text color
 
 ---
 
-Placeholder distractor 2
+Adjusting font size
 
 ---
 
-Placeholder distractor 3
+Adding animations
 
 #### --answer--
 
-Placeholder answer
+Creating responsive layouts
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which attribute helps improve accessibility for links?
 
 #### --distractors--
 
-Placeholder distractor 1
+href
 
 ---
 
-Placeholder distractor 2
+target
 
 ---
 
-Placeholder distractor 3
+class
 
 #### --answer--
 
-Placeholder answer
+title
```

Comment:
This is a very controversial answer and I have a feeling most accessibility professionals would not agree. The title attribute has several accessibility issues, such as people who use the keyboard only (which includes screen reader users) will not be able to activate the tooltip. People using touch devices may also have problems activating the tooltip. In general, it is best to avoid the title attribute except on iframes, where it is basically required. I don't think there is a good answer to this question and it should probably be removed.